src/frontend/templates/header.html

```diff
@@ -46,11 +46,6 @@
 
 <body>
     <header>
-        <div class="navbar">
-            <div class="container d-flex justify-content-center">
-                <div class="h-free-shipping">Free shipping with $75 purchase!</div>
-            </div>
-        </div>
         <div class="navbar sub-navbar">
```

Comment:
Is the `navbar` on this div referring to another class? Or is it the same as the one removed above?

Reply:
The `navbar` that is removed in the CSS file should only be used by the shipping fee banner. The `"navbar sub-navbar"` has specific styles in CSS